src/main/kotlin/no/nav/familie/tilbake/behandling/steg/IBehandlingssteg.kt

```diff
@@ -1,11 +1,19 @@
 package no.nav.familie.tilbake.behandling.steg
 
+import no.nav.familie.tilbake.api.dto.BehandlingsstegDto
 import no.nav.familie.tilbake.behandlingskontroll.domain.Behandlingssteg
+import no.nav.familie.tilbake.common.exceptionhandler.Feil
 import java.util.UUID
 
 interface IBehandlingssteg {
 
-    fun utførSteg(behandlingId: UUID)
+    fun utførSteg(behandlingId: UUID) {
+        throw Feil(message = "Noe gikk galt,er i default method implementasjon for $behandlingId")
```

Comment:
"Implementasjon mangler"  i stedet  for "Noe gikk galt", kanskje. Mellomrom etter komma.